cmd/kube-apiserver/app/testing/testdata/10.0.1.10_10.0.0.1_kubernetes.default.svc-kubernetes.default-kubernetes-localhost.crt

```diff
@@ -0,0 +1,40 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I think that this certificates got leaked from your testing :smile:

Reply:
Oops, removed!